libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -28,4 +28,84 @@
     #define IotLogDebugWithArgs( format, ... )
 #endif /* ifdef USE_AWS_IOT_CSDK_LOGGING */
 
+/**
+ * @brief The HTTP protocol version of this library is HTTP/1.1.
+ */
+#define HTTP_PROTOCOL_VERSION              "HTTP/1.1"
+
+/**
+ * @brief Default value when pRequestInfo->pPath == NULL.
+ */
+#define HTTP_EMPTY_PATH                    "/"
+#define HTTP_EMPTY_PATH_LEN                ( uint8_t ) ( sizeof( HTTP_EMPTY_PATH ) - 1 )
```

Comment:
You can remove the `(uint8_t)` typecasts from the `sizeof()` calls as that will raise compiler warnings of reducing type width. (I know that I had included the typecast in a different PR's comment, my bad)

Reply:
No worries! Search and replace to the rescue.